src/install-monerod-in-termux.sh

```diff
@@ -191,14 +199,14 @@ cd $NODE_CONFIG
 # Unrestricted RPC binds
 	rpc-bind-ip=127.0.0.1         # Bind to local interface. Default = 127.0.0.1
 	rpc-bind-port=18081           # Default = 18081
-	#confirm-external-bind=1       # Open node (confirm). Required if binding outside of localhost
-	#restricted-rpc=1              # Prevent unsafe RPC calls.
+	#confirm-external-bind=1      # Open node (confirm). Required if binding outside of localhost
+	#restricted-rpc=1             # Prevent unsafe RPC calls.
 
 # Services
 	rpc-ssl=autodetect
   	no-zmq=1
-	no-igd=1                         # Disable UPnP port mapping
-	db-sync-mode=safe                # Slow but reliable db writes
+	no-igd=1                            # Disable UPnP port mapping
+	db-sync-mode=fast:async:1000000     # Switch to db-sync-mode=safe for slow but more reliable db writes
```

Comment:
hyc's recommendation to use fast:sync instead of slow

Reply:
did he suggest fast:sync or fast:async

Reply:
fast:async 👍
I used the parameters as advised

"completely async, never explicit flush
currently there is no db-sync-mode setting that allows that, best you can do is fast:async:1000000 (some large number of blocks between flushes)"